Common/Commands/QueuedCommands.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace QuantConnect.Commands
+{
+    /// <summary>
+    /// Represent a queue of ICommand that can also be serialized or deserialized to xml
+    /// http://www.codeproject.com/Articles/738100/XmlSerializer-Serializing-list-of-interfaces
+    /// </summary>
+    public class QueuedCommands : Queue<ICommand>, IXmlSerializable
```

Comment:
Hi mchandschuh,

I've tried to serialize the Queue<ICommand> directly but the XMLSerializer does not allow to serialize interfaces which makes sense. I have found this work around on the web to be able to serialize the queue. 

I have no objection to rename it to CommandQueue

Thanks,